precode.sh

```diff
@@ -6,38 +6,42 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p task/dir1 task/dir2 task/dir3 task/dir3/dir4
 # изменяем текущую директорию на task
-
+cd task
 # создаём пустой файл task/dir2/empty
-
+touch ./dir2/empty
 # создаём файл task/dir2/hello.sh с таким содержанием:
 # #!/bin/bash
 # echo "$1, привет!"
-
+mkdir -p /task/dir2
+echo '#!/bin/bash' > /task/dir2/hello.sh
+echo 'echo "$1, привет!"' >> /task/dir2/hello.sh
 # устанавливаем для task/dir2/hello.sh права rwxrw-r--
-
+chmod 764 /task/dir2/hello.sh
```

Comment:
Не нужно обращаться к корню файловой системы, мы уже в папке `task` и нужно строить путь от текущей директории
```suggestion
chmod 764 dir2/hello.sh
```

Reply:
Нужно исправить далее все пути